src/Views/gagepage.html

```diff
@@ -59,6 +59,10 @@ <h4 class="modal-title">StreamStats Data-Collection Station Report</h4>
                                 <th scope="row">Agency</th>
                                 <td>{{vm.gage.agency.name}}</td>
                             </tr>
+                            <tr>
+                               <th scope="row">NWIS Period of Record</th>
+                               <td>{{vm.gage.StartDate | date:'MM/dd/yyyy'}} - {{vm.gage.EndDate | date:'MM/dd/yyyy'}}</td>
```

Comment:
Put "Undefined" instead of "-" if there is no valid StartDate or EndDate

![image](https://user-images.githubusercontent.com/40237491/159062020-0e350cc1-d85c-4a59-859a-e0c2fee1cd9c.png)